app/src/features/apiClient/helpers/testGeneration/buildPostResponseTests.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ Example Generated Tests Block Structure (without start/end markers):
+
+ rq.test("Status code is 201", () => {
+   rq.response.to.have.status(201);
+ });
+
+ rq.test("Request method is POST", () => {
+   rq.expect(rq.request.method).to.equal("POST");
+ });
+
+ rq.test("Response has valid JSON body", () => {
+   rq.response.to.have.jsonBody();
+ });
+
+ rq.test("Response body is JSON object", () => {
+   const parsed = JSON.parse(rq.response.body);
+   rq.expect(parsed).to.be.an("object").and.not.to.be.an("array");
+ });
+*/
+
+interface BuildTestsOptions {
+  method?: string;
+  status?: number;
+  hasJsonObjectBody?: boolean;
+}
+
+export function hasAutogeneratedTests(script: string | undefined): boolean {
```

Comment:
Lets rename this to `hasTests` or something similar, since we no longer use marker.